tiatoolbox/models/engine/semantic_segmentor.py

```diff
@@ -440,7 +440,7 @@ def __init__(
         self.pretrained_model = pretrained_model
         self.batch_size = batch_size
         self.num_loader_workers = num_loader_workers
-        self.num_postproc_workers = None
+        self.num_postproc_workers = num_postproc_workers
         self.verbose = verbose
         self.auto_generate_mask = auto_generate_mask
```

Comment:
@simongraham set to None

Reply:
We force it, always.  `num_postproc_workers` argument is for compatibility with subclasses (which may actually have post proc, and they will need to set it posthoc (overwriting) ). I dont think deepsource will complain.

Reply:
It does complain - it says unused argument if I do this

Reply:
You are welcome to try if you like and make a push

Reply:
@vqdang  https://deepsource.io/gh/TissueImageAnalytics/tiatoolbox/run/1b7bea27-6bb5-4cf5-b1e5-101652aec4aa/python/PYL-W0613